scripts/remote-dev-codex-runtime.py

```
@@ -483,16 +662,22 @@ def candidate_run(
-        env=candidate_env(home),
-        stdin=subprocess.DEVNULL,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        bufsize=0,
-        preexec_fn=drop_privileges if os.name == "posix" else None,
-    )
+    require_candidate_path(Path(argv[0]), executable=True)
+    require_candidate_path(cwd)
+    require_candidate_path(home)
+    try:
+        process = subprocess.Popen(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)